src/transformers/agents/python_interpreter.py

```diff
@@ -75,8 +85,8 @@ def get_iterable(obj):
         raise InterpreterError("Object is not iterable")
 
 
-def evaluate_unaryop(expression, state, tools):
-    operand = evaluate_ast(expression.operand, state, tools)
+def evaluate_unaryop(expression, state, static_tools, custom_tools):
```

Comment:
We add the distinction between `static_tools` and `custom_tools` everywhere: `static_tools` are the ones that the interpreter is not allowed to overwrite or create new ones (like base Python functions or user-defined tools), whereas `custom_tools` are the ones it can modify or extend.

Reply:
Ok, understood! Can you clarify that in the methods' docstrings?